data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -60,119 +62,213 @@
+    field_names.PM25_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.PM25_EXPOSURE_LOW_INCOME_FIELD,
+    field_names.DIESEL_FIELD,
+    field_names.DIESEL_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.DIESEL_PARTICULATE_MATTER_LOW_INCOME_FIELD,
+    field_names.TRAFFIC_FIELD,
+    field_names.TRAFFIC_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.TRAFFIC_PROXIMITY_LOW_INCOME_FIELD,
+    field_names.HOUSING_BURDEN_FIELD,
+    field_names.HOUSING_BURDEN_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.HOUSING_BURDEN_LOW_INCOME_FIELD,
+    field_names.LEAD_PAINT_FIELD,
+    field_names.LEAD_PAINT_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.LEAD_PAINT_MEDIAN_HOUSE_VALUE_LOW_INCOME_FIELD,
+    field_names.MEDIAN_HOUSE_VALUE_FIELD,
+    field_names.MEDIAN_HOUSE_VALUE_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.TSDF_FIELD,
+    field_names.TSDF_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.HAZARDOUS_WASTE_LOW_INCOME_FIELD,
+    field_names.NPL_FIELD,
+    field_names.NPL_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.SUPERFUND_LOW_INCOME_FIELD,
+    field_names.RMP_FIELD,
+    field_names.RMP_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.RMP_LOW_INCOME_FIELD,
+    field_names.WASTEWATER_FIELD,
+    field_names.WASTEWATER_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.WASTEWATER_DISCHARGE_LOW_INCOME_FIELD,
+    field_names.ASTHMA_FIELD,
+    field_names.ASTHMA_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.ASTHMA_LOW_INCOME_FIELD,
+    field_names.DIABETES_FIELD,
+    field_names.DIABETES_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.DIABETES_LOW_INCOME_FIELD,
+    field_names.HEART_DISEASE_FIELD,
+    field_names.HEART_DISEASE_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.HEART_DISEASE_LOW_INCOME_FIELD,
     field_names.LIFE_EXPECTANCY_FIELD,
     field_names.LOW_LIFE_EXPECTANCY_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
-    *DOWNLOADABLE_SCORE_INDICATOR_COLUMNS_FULL,
+    field_names.LOW_LIFE_EXPECTANCY_LOW_INCOME_FIELD,
+    field_names.MEDIAN_INCOME_AS_PERCENT_OF_AMI_FIELD,
+    field_names.LOW_MEDIAN_INCOME_AS_PERCENT_OF_AMI_FIELD
+    + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.LOW_MEDIAN_INCOME_LOW_HS_EDUCATION_FIELD,
+    field_names.LINGUISTIC_ISO_FIELD,
+    field_names.LINGUISTIC_ISOLATION_LOW_HS_EDUCATION_FIELD,
+    field_names.UNEMPLOYMENT_FIELD,
+    field_names.LINGUISTIC_ISO_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.UNEMPLOYMENT_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.POVERTY_LESS_THAN_100_FPL_FIELD,
+    field_names.POVERTY_LESS_THAN_100_FPL_FIELD
+    + field_names.PERCENTILE_FIELD_SUFFIX,
+    field_names.POVERTY_LOW_HS_EDUCATION_FIELD,
+    field_names.HIGH_SCHOOL_ED_FIELD,
+    field_names.HIGH_SCHOOL_ED_FIELD + field_names.PERCENTILE_FIELD_SUFFIX,
```

Comment:
Nit: I don't think we need `field_names.HIGH_SCHOOL_ED_FIELD + field_names.PERCENTILE_FIELD_SUFFIX`, that's from my hot pink in row 70 – but it also doesn't hurt to keep it.

Reply:
your pink says to have both in the downloadable

Reply:
(and to remove the percentile from the tiles, which ive done)

Reply:
oh ok! whoops. don't listen to me, listen to other me.